apps/docs/src/app/core/component-docs/resizable-card-layout/examples/resizable-card-layout-example.component.scss

```diff
@@ -0,0 +1,8 @@
+.fd-docs-cursor {
+    cursor: auto !important;
```

Comment:
why do you need to change the cursor? This is a documentation page, you only add styling for representational purposes

Reply:
removed now. because of using popover, cursor changing to "grab".